src/components/Reactions/AddReactionBubble.js

```diff
@@ -68,12 +69,17 @@ const AddReactionBubble = (props) => {
 
     return (
         <Tooltip text={props.translate('emojiReactions.addReactionTooltip')}>
-            <Pressable
+            <PressableWithFeedback
                 ref={ref}
                 style={({hovered, pressed}) => [styles.emojiReactionBubble, styles.userSelectNone, StyleUtils.getEmojiReactionBubbleStyle(hovered || pressed, false, props.isContextMenu)]}
                 onPress={Session.checkIfActionIsAllowed(onPress)}
                 // Prevent text input blur when Add reaction is clicked
                 onMouseDown={(e) => e.preventDefault()}
+                accessibilityLabel={props.translate('emojiReactions.addReactionTooltip')}
+                accessibilityRole="button"
+                // disable dimming
+                hoverDimmingValue={1}
```

Comment:
We had a slack conversion for providing hoverDimmingValue from parent here - https://expensify.slack.com/archives/C01GTK53T8Q/p1686151320529999?thread_ts=1685741951.521769&cid=C01GTK53T8Q

Will confirm on the same before approving.

Reply:
It seems like we will be setting hoverDimmingValue as 1 as default. But that change is in PR. 
Anyways i am fine with this for now. Since its not wrong to pass default value again as well(just not required) and I don't think we need to block this PR for the defaultValue change as well. Correct me if wrong @luacmartins 

https://github.com/Expensify/App/pull/19882/files#diff-4c815c48cfd171056c4be0e2bc556302e115c20c2bef5b5b00dbe55450faca5aR142